src/test/java/org/openrewrite/java/migrate/Java8ToJava11Test.java

```diff
@@ -105,4 +122,40 @@ void bar() {
           )
         );
     }
+
+    @Test
+    void Krb5LoginModuleTest() {
```

Comment:
These tests don't appear to be related to your pull request. Perhaps they were merged by mistake